cpp/CMakeLists.txt

```diff
@@ -105,3 +105,11 @@ target_link_libraries(ParquetSharpNative
 add_definitions(-DARROW_STATIC)
 add_definitions(-DARROW_NO_DEPRECATED_API)
 add_definitions(-DPARQUET_STATIC)
+
+if (UNIX AND CMAKE_BUILD_TYPE STREQUAL Release)
+  if (CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
+    set(STRIP_OPTIONS -x)
```

Comment:
Not great that the `-s` lives in the mainfile while this is her. Probably best to move the `-s` here.

Reply:
See explanation above.